src/utils/execAsync.ts

```diff
@@ -29,13 +29,21 @@ export async function execAsync(command: string, options?: cp.ExecOptions & { st
     if (progress) {
         stdoutIntermediate = new stream.PassThrough();
         stdoutIntermediate.on('data', (chunk: Buffer) => {
-            progress(bufferToString(chunk), false);
+            try {

```

Comment:
A sort of defense-in-depth; in case `progress()` decides to throw, it shouldn't get thrown out of this callback handler.